.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 /test
 /tmp
+coverage/
```

Comment:
You asked to do it Friday :) https://github.com/Zenika/react-workbench/issues/58